core/src/test/java/org/frankframework/senders/JavascriptSenderTest.java

```diff
@@ -255,33 +261,35 @@ public void errorInJavascriptCode(JavaScriptEngines engine) throws Configuration
 	//The input is expected to be of type integer but an input of type Sting is given.
 	@ParameterizedTest
 	@EnumSource(JavaScriptEngines.class)
-	public void wrongInputAsFirstParameter(JavaScriptEngines engine) throws ConfigurationException, SenderException, TimeoutException, IOException {
-
-		Message input = new Message("Stringinput");
+	void wrongInputAsFirstParameter(JavaScriptEngines engine) throws ConfigurationException, SenderException {
+		dummyInput = new Message("Stringinput");
 		sender.setJsFileName("Javascript/JavascriptTest.js");
 		sender.setJsFunctionName("f2");
 		sender.setEngineName(engine);
 
-		session.put("originalMessage", input);
+		session.put("originalMessage", dummyInput);
 
 		sender.addParameter(ParameterBuilder.create().withName("x").withSessionKey("originalMessage").withType(ParameterType.INTEGER));
 		sender.addParameter(ParameterBuilder.create("y", "2").withType(ParameterType.INTEGER));
 
 		sender.configure();
 		sender.open();
 
-		assertThrows(SenderException.class, () -> sender.sendMessageOrThrow(input, session));
+		assertThrows(SenderException.class, () -> sender.sendMessageOrThrow(dummyInput, session));
 	}
 
 	static Stream<Arguments> testWithJavaScriptReturningAnObject() {
 		return Stream.of(
 				Arguments.of(JavaScriptEngines.J2V8, "returnObject", "[object Object]"),
-				Arguments.of(JavaScriptEngines.J2V8, "returnArray", "1,3,5")
+				Arguments.of(JavaScriptEngines.J2V8, "returnArray", "1,3,5"),
+				Arguments.of(JavaScriptEngines.GRAALJS, "returnObject", "{answer: 42}"), // Don't understand why this is different in GraalJS
+				Arguments.of(JavaScriptEngines.GRAALJS, "returnArray", "(3)[1, 3, 5]")
```

Comment:
De bovenste is goed, het is fijn dat hij nu Json responses terug kan geven. Maar de array had ik anders verwacht...
Zou het mogelijk zijn die `(3)` er van af te snoepen en iets terug te geven waar de eindgebruiker wat meer mee kan?

Reply:
Ik vraag me af of die `(3)` gerelateerd is aan de lengte van de array?

Reply:
Ja, dat is hij zeker. Ik heb ook de toString gevonden in de Javascript engine die dit doet. Ik ben bezig geweest met type mappers om te zorgen dat hij zich anders gedraagt. Helaas werd dat nog niet goed opgepakt.